src/main/java/frc/robot/subsystems/IndexerSubsystem.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkLimitSwitch;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.IndexerConstants;
+
+public class IndexerSubsystem extends SubsystemBase {
+
+	private CANSparkMax m_indexerMotor;
+	private SparkLimitSwitch m_forwardLimitSwitch;
+
+	/**
+	 * Creates a new IndexerSubsystem.
+	 * 
+	 * @throws Exception
+	 */
+	public IndexerSubsystem() {
+		m_indexerMotor = new CANSparkMax(IndexerConstants.kIndexerPort, IndexerConstants.kIndexerMotorType);
```

Comment:
```suggestion
		m_indexerMotor = new CANSparkMax(IndexerConstants.kIndexerPort, MotorType.kBrushless);
```
Make sure to add MotorType as an import.